Descent3/init.cpp

```diff
@@ -1457,6 +1460,8 @@ void LoadGameSettings() {
   Database->read("RS_bitdepth", &Render_preferred_bitdepth, sizeof(Render_preferred_bitdepth));
   Database->read_int("RS_resolution", &Game_video_resolution);
   Database->read_int("RS_bilear", &Render_preferred_state.filtering);
+  Database->read_int("RS_windowwidth", &Game_window_res_width);
+  Database->read_int("RS_windowheight", &Game_window_res_height);
```

Comment:
No need to read "RS_fullscreen" now ?

Reply:
RS_fullscreen is read in winmain.cpp, it needs to be read before the initial window is created